### PR TITLE
Smart module resolve from within lambdaws

### PR DESCRIPTION
**Pull request by slvnperron**

### CODE REVIEW

File: lib/lambdaws.js [hunk 1]
```diff
@@ -21,11 +22,32 @@ var _sqsHelper = null;
...
+        // For built in or global modules
+        return require.resolve(p);
+    }
+    catch(ex1) { // Not found
+        var fullPath = path.join(relativeDir, p);
+        try {
+            return require.resolve(fullPath);
+        }
+        catch(ex2) { // Not found
+            throw "Could not resolve module [" + p + "] from [" + relativeDir + "] at [" + fullPath + "]";
```
Comment (EFF) [inline lib/lambdaws.js]: throw Error("your error message") if we want good stack trace

**Comment by EFF** [comment]
Love it, so simple !

**Comment by slvnperron** [comment]
Thanks. Can I merge yet?

**Comment by slvnperron** [comment]
I added smart resolve for inlined functions following PR #26

**Comment by EFF** [comment]
@slvnperron Yes sorry thought "love it" meant merge it... LGTM then !

**Comment by slvnperron** [comment]
Sorry will know in the future haha. Used to lgtm's

**Pull request merged**